sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -449,7 +457,7 @@ public abstract static class DataSourceConfiguration implements Serializable {
     abstract @Nullable ValueProvider<Integer> getMaxConnections();
 
     @Pure
-    abstract @Nullable ClassLoader getDriverClassLoader();
+    abstract @Nullable ValueProvider<String> getDriverJarDirectory();
```

Comment:
We cannot remove this getDriverClassLoader() due to backwards compatibility. Can we just add the new method instead.

Reply:
Added it back, but we should be able to deprecate it since I don't think its usable